components/prism-bqn.js

```diff
@@ -0,0 +1,63 @@
+Prism.languages.bqn = {
+	'shebang': {
+		pattern: /^#![ \t]*\/.*/,
+		alias: 'important',
+		greedy: true
+	},
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'string-literal': {
+		pattern: /"(?:[^"]|"")*"/,
+		greedy: true,
+		alias: 'string'
+	},
+	'character-literal': {
+		pattern: /'(?:[\s\S]|[\uD800-\uDBFF][\uDC00-\uDFFF])'/,
+		greedy: true,
+		alias: 'char'
+	},
+	'function': /•[\w¯.∞π]+[\w¯.∞π]*/,
+	'dot-notation-on-brackets': {
+		pattern: /\{(?=.*\}\.)|\}\./,
+		alias: 'namespace'
+	},
+	'special-name': {
+		pattern: /(?:𝕨|𝕩|𝕗|𝕘|𝕤|𝕣|𝕎|𝕏|𝔽|𝔾|𝕊|_𝕣_|_𝕣)/,
+		alias: 'keyword'
+	},
+	'dot-notation-on-name': {
+		pattern: /[A-Z_][\w¯∞π]*\./i,
```

Comment:
Is Π invalid here as well?

Reply:
Indeed, sorry. I should have noticed that; I keep forgetting it is a letter.

Reply:
Pushed in most recent commit.